websites/S/Streamlabs/presence.ts

```diff
@@ -82,119 +189,21 @@ presence.on("UpdateData", async () => {
 					presenceData.smallImageText = strings.pause;
 					presenceData.endTimestamp = null;
 					break;
+				default:
+					presenceData.smallImageKey = null;
+					presenceData.smallImageText = null;
+					break;
 			}
 			presenceData.details = `Watching ${clipTitle[0]} to`;
```

Comment:
I know this PR didn't touch this line but it sounds weird, do you have a screenshot of this in action?